benchmarks/index.js

```diff
@@ -73,6 +73,20 @@ suite
     },
     onComplete: resetTestState,
   })
+  .add('count without filter', {
+    defer: true,
+    fn: function(deferred) {
+      Todo.count(function() {
+        deferred.resolve();
+      });
+    },
+    onStart: function() {
+      const todos = [];
+      for (let i = 0; i < 1000; i++)
```

Comment:
nitpick: let's include the curly braces for this `for` loop, even though it's one line.

Reply:
Done